cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -365,3 +366,61 @@ def _cleanup():
         cleanup_target(provider, migrated_vm)
 
     assert src_vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.tier(2)
+def test_osp_if_no_password_is_exposed_in_logs_during_migration(appliance, provider, request,
+                                                                mapping_data_vm_obj_mini):
+    """
+    title: OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: sshveta
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for Vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2.
+            3. logs should not show password during migration
+    """
+    cred = []
+    cred.append(credentials["vsphere6-nested"]["password"])
+    cred.append(credentials["vmware-ssh-key"]["password"])
+    cred.append(credentials["host_default"]["password"])  # same for both RHV and OSP
+    cred.append(credentials["cloudqe-osp13-ims-rsa"]["password"])
+
+    migration_plan_collection = appliance.collections.v2v_migration_plans
+    migration_plan = migration_plan_collection.create(
+        name=fauxfactory.gen_alphanumeric(start="plan_"),
+        description=fauxfactory.gen_alphanumeric(start="plan_desc_"),
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        vm_list=mapping_data_vm_obj_mini.vm_list,
+        target_provider=provider
+    )
+
+    assert migration_plan.wait_for_state("Started")
+    assert migration_plan.wait_for_state("In_Progress")
+    assert migration_plan.wait_for_state("Completed")
+    assert migration_plan.wait_for_state("Successful")
+
+    # Check log files for any exposed password
+    evm_log = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; grep -inr password log/evm.log").output
+    automation_log = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; grep -inr password log/automation.log").output
```

Comment:
@mnadeem92 I recommend using the `LogValidator` for this and then using a fixture to search for any passwords. I implemented something similar in the auth tests with the `log_monitor` fixture. 

https://github.com/ManageIQ/integration_tests/blob/0f6c358b7a91a2beb9fda3cec343b8ae8f167059/cfme/tests/integration/test_cfme_auth.py#L124-L132

Note that for you the failure patterns will be the `cred` list you've defined. 

Any test that uses the fixture can then do 
```python,
    # assert no pwd in logs
    assert log_monitor.validate()
```
at the end of the test